.github/ISSUE_TEMPLATE/issue report.yml

```diff
@@ -1,59 +1,135 @@
-        If you are using an old release of Jellyfin, please also explain why.
+        After upgrading to version x.y.z of Jellyfin, the "login disclaimer" is showing incorrect text. It appears to me that it is appending the server name to the end of the login disclaimer, and showing that to a user. It might be a regression from pull request x. I have tried rebooting my host as well as my container multiple times. I tested this functionality on different clients, and it happens to all the tested clients (client x, y, z), that support the login disclaimer functionality. This makes me believe it is a server side issue.
     validations:
       required: true
   - type: textarea
     id: repro-steps
     attributes:
-      label: Reproduction Steps
+      label: Reproduction steps
+      description: Reproduction steps should be complete and self-contained. Anyone can reproduce this issue by following these steps. Furthermore, the steps should be clear and easy to follow.
       placeholder: |
-        1. In this environment...
-        2. With this config...
-        3. Run '...'
-        4. See error...
+        1. Sign in on the Jellyfin web client, with an admin account, using a browser of your choice.
+        2. Navigate to the dashboard.
+        3. Select "general".
+        4. Change the login disclaimer to something like "I am a cool disclaimer!"
+        5. Save the settings.
+        6. Sign out.
+        7. Make sure you are on the sign in screen. Otherwise, navigate to the sign in screen manually.
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: What is the current _bug_ behavior?
+      description: Write down the incorrect behavior that currently happens after following the reproduction steps.
+      placeholder: |
+        The login disclaimer on the sign in screen has the server name appended to the text. The text shown is: "I am a cool disclaimer!jellyfinserver".
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: What is the expected _correct_ behavior?
+      description: Write down the correct expected behavior that is supposed to happen after following the reproduction steps.
+      placeholder: |
+        The login disclaimer on the sign in screen should only show the configured text. The text that should be shown is: "I am a cool disclaimer!".
     validations:
       required: true
   - type: dropdown
     id: version
     attributes:
-      label: Jellyfin Version
-      description: What version of Jellyfin are you running?
+      label: Jellyfin Server version
+      description: What version of Jellyfin are you using?
       options:
-        - 10.9.0
-        - 10.8.13
-        - 10.8.12 or older (please specify)
-        - Weekly unstable (please specify)
-        - Master branch
+        - 10.9.6
+        - Master
+        - Unstable
+        - Older*
+      default: 0
```

Comment:
People should have to explicitly choose this otherwise they are likely to just leave the default.